### DISCUSSION TITLE
Top 10 OBS NDI Troubleshooting FAQ [VID]

### DISCUSSION THREAD

**Discussion by Helgaiden** [DistroAV/DistroAV#645] [General]
Hi again @Palakis and everyone else. Again, I don't really know where else to put this but i'll leave it here. I've been a big proponent of OBS NDI since doing our original vid on it over a year ago and have now taken the most common questions that were asked in the comments of that video and created a FAQ video that goes over many things. Many people here might find it helpful so i thought i'd post it. Thanks.

https://www.youtube.com/watch?v=N3KlbU5AYP0

**Comment by Palakis** [comment]
Thanks @Helgaiden! You can share this on the OBS' forums and Discord server.

I'll have to close this issue eventually. I guess I should put a list of video guides somewhere in the README file. I'll keep it open in the meantime.
